amazon/ionbenchmark/ion_load_dump.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import amazon.ion.simpleion as ion
+
+
+class IonLoadDump:
+    """
+    Wrapper for simpleion API that holds some configuration so that the configuration can be encapsulated here instead
+    of having to be plumbed through the whole benchmark code.
+
+    Results of profiling indicate that this adds a trivial amount of overhead, even for small data. If Ion Python
+    performance improves by >1000% from June 2023, then this may need to be re-evaluated.
+    """
+    def __init__(self, binary, c_ext=None):
```

Comment:
why not just default to `True`?

Reply:
![why not?](https://imgflip.com/i/7txp2a)